deployer/utils.py

```diff
@@ -32,3 +35,103 @@ def print_colour(msg: str, colour="green"):
     else:
         # colour not recognized, no colors
         print(msg)
+
+
+def create_markdown_comment(support_staging_matrix, prod_matrix):
+    """Convert a list of dictionaries into a Markdown formatted table for posting to
+    GitHub as comments. This function will write the Markdown content to a file to allow
+    a GitHub Actions to upload it as an artifact and reuse the content in another
+    workflow.
+
+    Args:
+        support_staging_matrix (list[dict]): The support of staging jobs to be converted
+            into a Markdown formatted table
+        prod_matrix (list[dict]): The production jobs to be converted into a Markdown
+            formatted table
+    """
+    # A dictionary to convert column names
+    column_converter = {
+        "cluster_name": "Cluster Name",
+        "provider": "Cloud Provider",
+        "upgrade_support": "Upgrade Support?",
+        "reason_for_support_redeploy": "Reason for Support Redeploy",
+        "upgrade_staging": "Upgrade Staging?",
+        "reason_for_staging_redeploy": "Reason for Staging Redeploy",
+        "hub_name": "Hub Name",
+        "reason_for_redeploy": "Reason for Redeploy",
+    }
+
+    # A dictionary to convert row values when they are Boolean
+    row_converter = {
```

Comment:
```suggestion
    boolean_converter = {
```

I feel like this is more descriptive for what it does?

Reply:
Agreed! Done in [`fe9be10` (#1424)](https://github.com/2i2c-org/infrastructure/pull/1424/commits/fe9be101f722c03e901e41bf5706fd1ff9adfa33)